### ISSUE TITLE
process project.toml as part of the tekton `buildpack` tasks

### ISSUE THREAD

**Issue by dmitry-mightydevops** [saritasa-nest/saritasa-devops-tekton#8]
Build processing of project.toml file env vars into our tekton solution
use this base public python image from our dockerhub: https://keys.saritasa.com/cred/detail/12938/
https://hub.docker.com/repository/docker/saritasallc/python3

if you want to adjust image (i.e. pip libs) - see my PR https://github.com/saritasa-nest/saritasa-devops-docker-images/pull/12 (see Makefile how to build the image)
(use full token from key 12938)

Use this
https://pypi.org/project/toml/

I already installed this package in the https://hub.docker.com/repository/docker/saritasallc/python3 image
```

In [1]: import toml

In [2]: toml_string = """
   ...: [[build.env]]
   ...: name =  "GOOGLE_RUNTIME_VERSION"
   ...: value = "3.9.5"
   ...: """

In [3]: parsed_toml = toml.loads(toml_string)

In [4]: parsed_toml
Out[4]: {'build': {'env': [{'name': 'GOOGLE_RUNTIME_VERSION', 'value': '3.9.5'}]}}

In [5]: parsed_toml['build']
Out[5]: {'env': [{'name': 'GOOGLE_RUNTIME_VERSION', 'value': '3.9.5'}]}

In [6]: parsed_toml['build']['env']
Out[6]: [{'name': 'GOOGLE_RUNTIME_VERSION', 'value': '3.9.5'}]
```

**Comment by dmitry-mightydevops** [comment]
See video with details: https://drive.google.com/file/d/1QzUeCxq1mcQYkHgGQb77V2R0speUHxSV/view?usp=sharing

**Comment by dmitry-mightydevops** [comment]
Pls do this for
- buildpack task
- buildpack-frontend
- buildpack-dotnet

**Comment by kseniyashaydurova** [comment]
Fixed and merged new fixes in `feature/0.27.0`
